fog/recovery_db_iface/src/lib.rs

```diff
@@ -37,8 +37,11 @@ pub struct IngressPublicKeyRecordFilters {
 }
 
 /// A generic error type for recovery db operations
-pub trait RecoveryDbError: Debug + Display + Send + Sync {}
-impl<T> RecoveryDbError for T where T: Debug + Display + Send + Sync {}
```

Comment:
I think this blanket impl was a bad idea, this was from my more noobie rust days